src/gp4prog/gp4prog.h

```diff
@@ -33,6 +33,12 @@
 
 #include <libusb-1.0/libusb.h>
 
+//Windows appears to define an ERROR macro in its headers.
+//Conflicts with ERROR enum defined in log.h.
+#if defined(_WIN32) && defined(ERROR)
+	#undef ERROR
+#endif
```

Comment:
`libusb` is including it, according to the output of the following command: `/C/msys64/mingw32/bin/g++.exe -H  -D__USE_MINGW_ANSI_STDIO=1  -Wall -Wextra -Wcast-align -Winit-self -Wmissing-declarations -Wswitch -Wwrite-strings -std=c++11  -shared-libgcc -ggdb -c /C/msys64/home/William/src/openfpga/src/gp4prog/main.cpp 2> main.e`